### PR TITLE
ready for review

### PR DESCRIPTION
**Pull request by skeydan**

### CODE REVIEW

File: _posts/2020-01-13-mixed-precision-training/mixed_precision.Rmd [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+
+# on first execution, this downloads the dataset
+imagenette <- tfds_load("imagenette/320px")
+
+# extract train and test parts
+train <- imagenette$train
+test <- imagenette$validation
+
+# batch size for the initial run
+batch_size <- 32
```
Comment (kevinykuo) [inline _posts/2020-01-13-mixed-precision-training/mixed_precision.Rmd]: Have you tried a bigger batch size? 32 might be too small to see any differences in performance.

File: _posts/2020-01-13-mixed-precision-training/mixed_precision.Rmd [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+model <- keras_model_sequential() %>% 
+  layer_conv_2d(filters = 32, kernel_size = 5, strides = 2, padding = "same", input_shape = c(256, 256, 3), activation = "relu") %>%
+  layer_batch_normalization() %>%
+  layer_conv_2d(filters = 64, kernel_size = 7, strides = 2, padding = "same", activation = "relu") %>%
+  layer_batch_normalization() %>%
+  layer_conv_2d(filters = 128, kernel_size = 11, strides = 2, padding = "same", activation = "relu") %>%
+  layer_batch_normalization() %>%
+  layer_global_average_pooling_2d() %>%
+  # separate logits from activations so actual outputs can be float32
+  layer_dense(units = 10) %>%
```
Comment (kevinykuo) [inline _posts/2020-01-13-mixed-precision-training/mixed_precision.Rmd]: Can we try a multiple of 8 for number of units?

File: _posts/2020-01-13-mixed-precision-training/mixed_precision.Rmd (hunk 1 above)
Comment (skeydan) [reply _posts/2020-01-13-mixed-precision-training/mixed_precision.Rmd]: thanks for commenting! I was trying batch sizes between and 32 and 512, for that reason...

File: _posts/2020-01-13-mixed-precision-training/mixed_precision.Rmd (hunk 2 above)
Comment (skeydan) [reply _posts/2020-01-13-mixed-precision-training/mixed_precision.Rmd]: there are 10 classes, so, no... also, I think given this is just a single layer after several conv layers, so even if this made a difference, it shouldn't be a big one...

btw I also tried without batchnorm, this didn't make a difference either

Comment (kevinykuo) [reply _posts/2020-01-13-mixed-precision-training/mixed_precision.Rmd]: LOL yeah this is the output layer, brainfart on my part

File: _posts/2020-01-13-mixed-precision-training/mixed_precision.Rmd (hunk 1 above)
Comment (kevinykuo) [reply _posts/2020-01-13-mixed-precision-training/mixed_precision.Rmd]: Oops I failed at RTFA, you're right :)
